EventFilter/Phase2TrackerRawToDigi/BuildFile.xml

```diff
@@ -1,13 +1,5 @@
-<use name="CondFormats/DataRecord"/>
-<use name="CondFormats/SiStripObjects"/>
-<use name="DataFormats/Common"/>
-<use name="DataFormats/DetId"/>
 <use name="DataFormats/FEDRawData"/>
-<use name="DataFormats/Phase2TrackerDigi"/>
```

Comment:
this is somewhat similar (used in the /plugins while the /plugins build file does not have this dependency explicitly.
At least in this case there was no recent PR that added this dep